Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/SyntaxReceiver.cs

```diff
@@ -61,12 +67,26 @@ public void OnVisitSyntaxNode(GeneratorSyntaxContext context)
             if (context.Node is ClassDeclarationSyntax classDeclarationSyntax && classDeclarationSyntax.AttributeLists.Count > 0)
             {
                 // Get the symbol being declared by the class, and keep it if its annotated
-                var methodSymbol = context.SemanticModel.GetDeclaredSymbol(classDeclarationSyntax);
+                var methodSymbol = ModelExtensions.GetDeclaredSymbol(
```

Comment:
`ModelExtensions.GetDeclaredSymbol` is a C# extension method for `SemanticModel`. Is the reason you switch this code to call the extension method directly through the extension method type holder?

Reply:
Nope, no good reason. Updated.